client/me/billing-history/index.jsx

```diff
@@ -69,10 +83,7 @@ module.exports = React.createClass( {
 					<BillingHistoryTable transactions={ data.billingHistory } />
 				</Card>
 
-				<SectionHeader label={ this.translate( 'Upcoming Charges' ) } />
-				<Card id="upcoming-charges">
-					<UpcomingChargesTable sites={ this.props.sites } transactions={ data.upcomingCharges } />
-				</Card>
+				{ this.renderUpcomingCharges( data ) }
```

Comment:
The code in `renderUpcomingCharges()` doesn't have much logic inside of it, and I think we prefer to keep the display code in `render()` as much as possible to keep things explicit. If the abstraction is necessary, it probably deserves its own React component, which I do no believe is the case here.

```js
render() {
	const hasBillingHistory = ! isEmpty( data.billingHistory );

	…

	{ hasBillingHistory &&
		<div>
			<SectionHeader … />
			<Card id="upcoming-changes">
				…
			</Card>
		</div> }
}
```

This makes it easier to see at a glance what will be rendered and keeps more consistency with other Calypso components.